ui-core/src/main/java/com/adyen/checkout/components/ui/view/AdyenSwipeToRevealLayout.kt

```diff
@@ -346,6 +349,20 @@ class AdyenSwipeToRevealLayout @JvmOverloads constructor(
         }
     }
 
+    override fun onAttachedToWindow() {
+        super.onAttachedToWindow()
+        fixElevationOverlay()
+    }
+
+    private fun fixElevationOverlay() {
```

Comment:
What exactly was this issue here?

Reply:
In dark mode elevation overlay makes surfaces lighter the higher the elevation (instead of casting a bigger shadow). For some reason the surface was way to light here, so this makes sure the surface has the same color as the surface behind this layout